.travis.yml

```diff
@@ -30,8 +30,8 @@ before_script: # configure a headless display to test plot generation
 script:
     - flake8 parcels
     - flake8 tests
-    - flake8 examples
-    - python parcels/scripts/pull_data.py
+    - flake8 parcels/examples
```

Comment:
What is the need/advantage of putting the `examples` directory into the `parcels` directory? The `tests` directory isn't there either, which is why originally we also had the `examples` directory in the root of the repo

Reply:
Let's discuss why I moved `include/` to `parcels/include/` (because you probably can see the problem on your own computer already):

<https://github.com/OceanParcels/parcels/blob/9b52f1f79dabd7d616adf9f2bfbceaa40e3d1f64/setup.py#L13> will install a _package_ called `include` and a package called `parcels`.  The resulting directory structure is

```bash
tree miniconda2/envs/py2_parcels/lib/python2.7/site-packages/{include/,parcels/}
```
```
miniconda2/envs/py2_parcels/lib/python2.7/site-packages/include
└── parcels.h
miniconda2/envs/py2_parcels/lib/python2.7/site-packages/parcels
├── codegenerator.py
├── codegenerator.pyc
├── compiler.py
├── compiler.pyc
├── field.py
├── field.pyc
├── fieldset.py
├── fieldset.pyc
├── __init__.py
├── __init__.pyc
├── kernel.py
├── kernel.pyc
├── kernels
│   ├── advection.py
│   ├── advection.pyc
│   ├── error.py
│   ├── error.pyc
│   ├── __init__.py
│   └── __init__.pyc
├── loggers.py
├── loggers.pyc
├── particlefile.py
├── particlefile.pyc
├── particle.py
├── particle.pyc
├── particleset.py
├── particleset.pyc
├── rng.py
├── rng.pyc
└── scripts
    ├── convert_IndexedOutputToArray.py
    ├── convert_IndexedOutputToArray.pyc
    ├── __init__.py
    ├── __init__.pyc
    ├── plotParticles.py
    ├── plotParticles.pyc
    ├── pull_data.py
    └── pull_data.pyc
```

But we want `include/` somewhere below `miniconda2/envs/py2_parcels/lib/python2.7/site-packages/parcels/`.  The same is true for `examples/` as soon as it is distributed with the package.

This was not a problem when setting `$PYTHONPATH` directly, because the `parcels.h` actually  was in `parcels/../include/`.

With <https://github.com/OceanParcels/parcels/pull/235/files#diff-2eeaed663bd0d25b7e608891384b7298R16>, I include, both, `include/` and `examples/` as `package_data` belonging to `parcels` rather than as a separate package.